src/caret_analyze/architecture/architecture.py

```diff
@@ -372,6 +387,19 @@ def update_message_context(self, node_name: str, context_type: str,
 
     def insert_publisher_callback(self, node_name: str,
                                   publish_topic_name: str, callback_name: str) -> None:
+        """
+        Insert callback to publisher in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node to insert publisher and callback
+        publish_topic_name : str
+            name of publish topic to insert publisher callback
+        callback_name : str
+            name of callback to insert publisher
```

Comment:
name of callback to be inserted for publisher?

Reply:
Fix it in [474c2bc](https://github.com/tier4/CARET_analyze/pull/300/commits/474c2bcf9fff70e85d47dc912f091e3c53122542).